### PR TITLE
Add examples of load balancing

### PR DESCRIPTION
**Pull request by interludic2000**

### CODE REVIEW

File: content/topic-guides/load-balancer/Device-based-origin.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+---
+title: Load Balancing - Device based origin selection
```
Comment (jstangroome) [inline content/topic-guides/load-balancer/Device-based-origin.md]: Choosing an origin based on device? Seems to be unrelated to balancing load.

File: content/topic-guides/load-balancer/Device-based-origin.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+In your `default.vcl` file you can set a header that's recognizable in section.io's `last-proxy` to determine which origin address and host header to make the request with. So you are going to need to define that origin first in your `section.config.json` file located in the root of your applications git repo. You will be adding a key called `alternate_origins` under `environments.Production`.
+
+#### `section.config.json`
+    
+    ...
+    "environments": {
+        "Production": {
+            "origin": {
+                "address": "100.100.100.100"
```
Comment (jstangroome) [inline content/topic-guides/load-balancer/Device-based-origin.md]: RFC 5737 defines IP addresses reserved for use in documentation:
https://tools.ietf.org/html/rfc5737#section-3

File: content/topic-guides/load-balancer/Device-based-origin.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    ...
+
+#### `default.vcl`
+
+Next we tell Varnish Cache to use this origin for requests if we detect the User-Agent as a mobile device. Use the origin names defined in `section.config.json`.
+
+    sub devicedetect {
+        # The VCL here is just an example, it is not a exhaustive device detection logic.
+        if (req.http.User-Agent ~ "(?i)samsung|iphone|ipod|pixel") {
+            set req.http.x-device-type = "mobile";
```
Comment (jstangroome) [inline content/topic-guides/load-balancer/Device-based-origin.md]: Perhaps avoid the `x-` prefix in our example since RFC 6648 deprecated that:
https://tools.ietf.org/html/rfc6648

File: content/topic-guides/load-balancer/Static-asset-origin.md [hunk 4]
```diff
@@ -1,78 +1,58 @@
...
+#### `section.config.json`
+    
+    ...
+    "environments": {
+        "Production": {
+            "origin": {
+                "address": "100.100.100.100"
+            },
+            "alternate_origins": {
+                "assets_example_com_100_100_100_120": {
```
Comment (jstangroome) [inline content/topic-guides/load-balancer/Static-asset-origin.md]: The domain name and IP address in the origin name is a legacy of V2 puppet code generation. Let's not encourage it further.

File: content/topic-guides/load-balancer/Device-based-origin.md (hunk 1 above)
Comment (interludic2000) [reply content/topic-guides/load-balancer/Device-based-origin.md]: I was working off the examples given in the current Docs
https://www.section.io/docs/topic-guides/load-balancer/

File: content/topic-guides/load-balancer/Device-based-origin.md (hunk 2 above)
Comment (interludic2000) [reply content/topic-guides/load-balancer/Device-based-origin.md]: Again, using the example IP address from current docs https://www.section.io/docs/topic-guides/load-balancer/

But happy to change them to align with RFC

File: content/topic-guides/load-balancer/Device-based-origin.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+    sub vcl_recv {
+        ...
+        call devicedetect;
+        if ( req.http.device-type == "mobile" ) {
+            set req.http.section-origin = "mobile_device";
+        } 
+        ...
+    }
+
+*Note 1:* Requests from non-mobile devices will be directed at the default origin of `100.100.100.100`.
```
Comment (jstangroome) [inline content/topic-guides/load-balancer/Device-based-origin.md]: missed an IP

File: content/topic-guides/load-balancer/GEO-based-origin.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+        ...
+        if ( req.http.section-io-geo-country-code ~ "(?i)^(US|CA)$" ) {
+            set req.http.section-origin = "US_origin";
+        } elsif ( req.http.section-io-geo-country-code ~ "(?i)^(AU|NZ)$" ) {
+            set req.http.section-origin = "AU_origin";
+        }
+        ...
+    }
+
+*Note 1:* Requests not from US/CA/AU/NZ will be directed at the default origin of `100.100.100.100`.
```
Comment (jstangroome) [inline content/topic-guides/load-balancer/GEO-based-origin.md]: another IP
